tests/sandbox/create_message_batches/test_success.py

```diff
@@ -69,7 +93,15 @@ def test_201_message_batch_valid_routing_plan_id(nhsd_apim_proxy_url, routing_pl
 @pytest.mark.sandboxtest
 def test_201_message_batch_valid_nhs_number(nhsd_apim_proxy_url):
     """
-    .. py:function:: Test valid NHS numbers
+    .. py:function:: Scenario: A user submitting a request with a \
+        valid NHS number receives a 201 response
+
+        | **Given** the user provides a valid NHS number
```

Comment:
**Given** the user provides a valid NHS number for a recipient in their new message batch